assets/scss/modules/_navigation.scss

```diff
@@ -147,3 +145,146 @@ a.back-to-top-link {
   }
 
 }
+
+/*
+Page Navigation
+
+A page navigation element with a previous and next link.
+
+Markup:
+<nav role="navigation" aria-label="Page navigation">
```

Comment:
Yeah, I had a read of https://www.paciellogroup.com/blog/2012/06/html5-accessibility-chops-when-to-use-an-aria-role/. It seems it wouldn't harm but is perhaps somewhat unnecessary.